doc/dev-guide/src/linting.md

```diff
@@ -0,0 +1,33 @@
+# Linting
+
+## Manual Linting
```

Comment:
The key is to keep the style consistent with the rest of the Rustup documentation.
```suggestion
## Manual linting
```